setup.cfg

```diff
@@ -1,6 +1,6 @@
 [metadata]
 name = krb5
-version = 0.6.0
+version = 0.6.0.post50
```

Comment:
Thanks, I keep on forgetting about bumping the version and keep on getting bit by this. Will need to look into the build process again to find a way to avoid this problem.

Reply:
as a workaround, pull requesters may follow PEP 440 convention in setup.cfg to avoid collisions with published release:) the pattern requirement is strict though...